app/code/Vladimirl/Chatter/view/frontend/templates/form_chatter.phtml

```diff
@@ -1,15 +1,22 @@
-<div>
-    <form id="vladimirl-chatter-form-chatter" method="post" autocomplete="off"
-          action="<?= $block->escapeUrl($block->getUrl('vladimrl_chatter_controller/submit/submit')) ?>"
-          data-mage-init='{"vladimirL_chatter_formChatter": {}}'
-          onsubmit="return false"
-    >
-        <div id="displayedForm"></div>
-        <textarea id="entryText" placeholder="Message" name="textRequest"
-               data-validate="{required:true}"></textarea>
-		<button type="submit" id="send-message-button">
-            <?= $block->escapeHtml(__('Send message')) ?></button>
-    <button type="button" id="close-chatter-batton">
-            <?= $block->escapeHtml(__('Close chat')) ?></button>
-	</form>	
-</div>
\ No newline at end of file
+<?php /** @var \Vladimirl\Chatter\Block\Data $block */ ?>
+<form id="vladimirl-chatter-form-chatter" method="post" autocomplete="off"
+      action="<?= $block->escapeUrl($block->getUrl('vladimrl_chatter_controller/submit/submit')) ?>"
+      data-mage-init='{"vladimirL_chatter_formChatter": {}}'
+      onsubmit="return false"
+>
+    <div id="messages-list">
+        <?php /** @var \Vladimirl\Chatter\Model\ChatMessage $massage */ ?>
+        <?php foreach ($block->getChatMessage() as $massage): ?>
+            <p><?= $massage; ?></p>
```

Comment:
@Vladimir-L No date, no author. It is not visually possible to distinguish customer and admin messages